kratos/processes/calculate_distance_to_skin_process.h

```diff
@@ -125,6 +125,17 @@ class KRATOS_API(KRATOS_CORE) CalculateDistanceToSkinProcess : public CalculateD
     ///@name Operations
     ///@{
 
+    /**
+     * @brief This method creates an pointer of the process
+     * @details We consider as input a Model and a set of Parameters for the sake of generality
+     * @param rModel The model to be consider
+     * @param ThisParameters The configuration parameters
+     */
```

Comment:
You can use [@copydoc](https://www.doxygen.nl/manual/commands.html#cmdcopydoc) instead of doing manual copy-pasta everywhere.

```suggestion
    /// @copydoc Process::Create
```

While you're at it, please fix the grammar as well.